db/01_database.sql

```diff
@@ -184,6 +184,40 @@ CREATE TABLE `rss` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `notifications`
+--
+
+DROP TABLE IF EXISTS `notifications`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `notifications` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `marker_id` varchar(38) COLLATE utf8_unicode_ci NOT NULL,
+  `principal` varchar(255) COLLATE utf8_unicode_ci NOT NULL,
```

Comment:
`principal` is a security term used in the `auth` table to cover authentication of users & non-users. For notifications & unsubscribed, I'd suggest using `username`.